index.template.html

```diff
@@ -3997,6 +3998,99 @@ <h3>Thing Template: Buzzer</h3>
+        This mapping is defined as a JSON-LD context, as explained later. Second, clients
+        can access the constraints defined on these IRIs by dereferencing them.
+        Constraints are defined as logical axioms in the RDF format, readily interpretable
+        by client programs.
+    </p>
+
+    <p>
+        All <a>Vocabulary Terms</a> referenced in <a href="#sec-vocabulary-definition"></a>
+        are serialized as (compact) JSON strings in a TD document. However, each of these
+        terms is unambiguously identified by a full IRI, as per the first Linked Data
+        principle [[LINKED-DATA]]. The mappings from JSON keys to IRIs is what the
+        <code>@context</code> value of a TD points to. For instance,
+    </p>
+
+    <p>
+        <code>https://www.w3.org/2019/wot/td/v1</code>
+    </p>
+
+    <p>
+        includes the following mappings (among others):
+        <table>
+            <tbody>
+                <tr>
+                    <td><code>properties</code></td>
+                    <td>→</td>
+                    <td><code>https://www.w3.org/2019/wot/td#hasPropertyAffordance</code></td>
+                </tr>
+                <tr>
+                    <td><code>object</code></td>
+                    <td>→</td>
+                    <td><code>https://www.w3.org/2019/wot/json-schema#ObjectSchema</code></td>
+                </tr>
+                <tr>
+                    <td><code>basic</code></td>
+                    <td>→</td>
+                    <td><code>https://www.w3.org/2019/wot/security#BasicSecurityScheme</code></td>
+                </tr>
+                <tr>
+                    <td><code>href</code></td>
+                    <td>→</td>
+                    <td><code>https://www.w3.org/2019/wot/hypermedia#hasTarget</code></td>
+                </tr>
+                <tr>
+                    <td>...</td>
+                </tr>
+            </tbody>
+        </table>
+    </p>
+
+    <p>
+        The context file in which these mappings are defined follows the JSON-LD 1.1 syntax [[JSON-LD11]].
+        Numerous JSON-LD libraries can automatically process the <code>@context</code> of a TD
+        and expand all the JSON strings it includes.
+    </p>
+
+    <p>
+        Once every <a>Vocabulary Term</a> of a TD is expanded to a IRI, the second step
+        consists in dereferencing this IRI to get fragments of the <a>TD Information Model</a>
+        that refer to that <a>Vocabulary Term</a>. For instance, the RDF document
+    </p>
+
+    <p>
+        <code>https://www.w3.org/2019/wot/json-schema#ObjectSchema</code>
+    </p>
+
+    <p>
+        states that the term <code>ObjectSchema</code> is a <a>Class</a> and more precisely,
+        a sub-class of <code>DataSchema</code>. Such logical axioms are represented in RDF
+        using formalisms of various complexity: here, sub-class relations are
+        expressed in the terms of RDF Schema [[RDF-SCHEMA]].
```

Comment:
it would be cool to see this triple in Turtle here:
`
https://www.w3.org/2019/wot/json-schema#ObjectSchema rdf:type owl:Class ;

                                            rdfs:subClassOf <https://www.w3.org/2019/wot/json-schema#DataSchema> ;

`

Reply:
See 0927457